client/signup/reskinned-processing-screen/style.scss

```diff
@@ -0,0 +1,48 @@
+@import '~@automattic/onboarding/styles/mixins';
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import 'landing/gutenboarding/onboarding-block/colors';
```

Comment:
Since this kind of use is a bit unexpected — I don't think we'd be able to each time test and avoid breakage on your flow by making changes to gutenboarding styles.

It would be better to move the whole loading animation component to `@automattic/onboarding`, and pull it from there to both Gutenboarding and here. What do you think? That'd make the boundary between these flows and style re-use much clearer.

Reply:
Thanks @simison. I removed the dependencies from the `style.scss` in the processing screen. The `mixins` would remain since it is already used in the `main` signup component. We also decided to leave the new processing screen in the current place for now because we may need to remove the component when the variation group loses the testing.